manifest.json

```diff
@@ -44,6 +44,14 @@
                 "default": "/agendav"
             },
             {
+		"name": "is_public",
```

Comment:
it makes no sense for this app to be "public", since it uses http_auth via ssowat to connect to each user's calendars.